CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Dev
```

Comment:
I would go ahead and just set this to either 1.2.8 (or, more semantically, 1.3.0 but cutting new minor versions is mildly annoying with our auto tagger releaser mechanism).

But anyway, mostly to not forget to do this

Reply:
Updated changelog to version this at 1.3.0. Now just to remind myself how to do the tagging to generate the new version. 🤔